src/views/ChatGpt/Message.tsx

```diff
@@ -21,11 +25,31 @@ const Message: React.FC<Props> = ({ message, onDelete }) => {
     y: -6,
   });
 
+  // https://github.com/remarkjs/react-markdown/issues/785#issuecomment-1966495891
+  const preprocessLaTeX = (content: string) => {
+    // Replace block-level LaTeX delimiters \[ \] with $$ $$
+    const blockProcessedContent = message.content.replace(
```

Comment:
You don't actually use the `content` parameter.

```suggestion
  const preprocessLaTeX = (content: string) => {
    // Replace block-level LaTeX delimiters \[ \] with $$ $$
    const blockProcessedContent = content.replace(
```

Also you can move this function outside of the component.